apps/rush-lib/src/cli/test/__snapshots__/CommandLineHelp.test.ts.snap

```diff
@@ -172,6 +175,20 @@ Optional arguments:
                         which projects were changed. If this parameter is not 
                         specified, the checked out branch is compared against 
                         the \\"master\\" branch.
+  --overwrite           If a changefile already exists, overwrite without 
+                        prompting (or erroring in --bulk mode).
+  --email EMAIL         The email address to use in changefiles. If this 
+                        parameter is not provided, the email address will be 
+                        detected or prompted for in intractive mode.
+  --bulk                If this flag is specified, apply the same change 
+                        message and bump type to all changed projects. The 
+                        --message and the --bump-type parameters must be 
+                        specified if the --bulk parameter is specified
+  --message MESSAGE     The message to apply to all changed projects if the 
+                        --bulk flag is provided.
+  --bump-type {major,minor,patch,none}
+                        The bump type to apply to all changed projects if the 
+                        --bulk flag is provided.
```

Comment:
Both `--message` and `--bump-type` seem like they would be useful for a non-bulk invocation as well, e.g. if you were invoking them from a script and didn't want interactive questions.  Is that out of scope for this PR?

Reply:
I think it's out-of-scope for this PR.

I'd argue that once you're using those parameters, you're not really using interactive mode anymore.